lexus_rx_450h_2019/drivers.launch.py

```diff
@@ -48,23 +49,45 @@ def generate_launch_description():
     declare_drivers_arg = DeclareLaunchArgument(
         name = 'drivers', default_value = 'dsrc_driver', description = "Desired drivers to launch specified by package name."
     )
+    ssc_package_name = LaunchConfiguration('ssc_package_name')
+    declare_ssc_package_name = DeclareLaunchArgument(
+        name = 'ssc_package_name', default_value = 'ssc_pm_lexus', description = "Third party ssc package to launch" 
+    )
```

Comment:
Is there value in this being an argument instead of hardcoded? Since this file is specific to the lexus it seems like it would never change